components/bl-granim-component/README.md

```diff
@@ -0,0 +1,97 @@
+| blendingMode            | *Select* <br> [`multiply`, `screen`, `normal`, `overlay`, `darken`, `lighten`, `lighter`, `color-dodge`, `color-burn`, `hard-light`, `soft-light`, `difference`, `exclusion`, `hue`, `saturation`, `color`, `luminosity`] | `multiply`    |       | NO           | YES        | Options for blending images with a gradient.                                                          |
+| isPausedWhenNotInView   | *Checkbox*                                                                                                                                                                                                                | `false`       |       | NO           | YES        | Stop the animation when it's not in window view.                                                      |
+| scrollDebounceThreshold | *Number*                                                                                                                                                                                                                  |               |       | NO           | YES        | The scroll debounce threshold (in ms).                                                                |
+| transitionSpeed         | *Number*                                                                                                                                                                                                                  |               |       | NO           | YES        | Transition duration between each gradient (in ms).                                                    |
+| loop                    | *Checkbox*                                                                                                                                                                                                                | `true`        |       | NO           | YES        | The animation loop.                                                                                   |
+
+## Events
+
+| Name               | Triggers                                   | Context Blocks                                                                                |
+|--------------------|--------------------------------------------|-----------------------------------------------------------------------------------------------|
+| On Start           | Triggered when the animation start.        |                                                                                               |
+| On Gradient Change | Triggered when a gradient change and loop. | ColorDetails: `{activeState: String, colorsFrom: Array, colorsTo: Array, isLooping: Boolean}` |
```

Comment:
isLooping -> loop

Also, I can't find any information about what is activeState